.github/workflows/daily-changelog-dates-trigger.yml

```
@@ -0,0 +1,76 @@
+name: daily-changelogs-dates-trigger
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 00 19 * * *
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  get-component-details:
+    name: Get component details from helm charts
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    environment: build-pipeline
+    outputs:
+      matrix: ${{ steps.component-details.outputs.matrix }}
+    steps:
+      - name: checkout docs repository
+        uses: actions/checkout@v4
+        with:
+          path: docs
+          fetch-depth: 1
+
+      - name: Creating github access token
+        id: token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ vars.APP_ID }}
+          private-key: ${{ secrets.APP_PRIVATE_KEY }}
+          owner: cumulocity-iot
+          repositories: c8y-iot-build-pipeline
```

Should allow us to avoid any changes in case of the next GitHub migration.
The `repositories` parameter is not required as the GitHub app we are using anyhow only has access to this single repo.
```suggestion
          owner: ${{ github.repository_owner }}
```